### PR TITLE
Prime: add knowledge to reverse frigate

### PR DESCRIPTION
**Pull request by JeffGainsNGames**

### CODE REVIEW

**Review by Samuel6710 — COMMENTED** [review]
I'd reword the change log entry to say "Reverse frigate now requires Knowledge (Beginner) in every room with conduits."

**Review by Samuel6710 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
+##### Tallon Overworld
+
+- Added: Reverse Frigate now requires Knowledge (Beginner) in every room where the door usually requires activating Thermal Conduits from the other side.
 - Changed: The Varia-only heat reduction is now done via a miscellaneous resource rather than being patched at runtime.
```
Comment (Miepee) [inline CHANGELOG.md:36]:
```suggestion
- Changed: The Varia-only heat reduction is now done via a miscellaneous resource rather than being patched at runtime.

##### Tallon Overworld

- Added: Reverse Frigate now requires Knowledge (Beginner) in every room where the door usually requires activating Thermal Conduits from the other side.
```

Comment (JeffGainsNGames) [reply CHANGELOG.md]: oh whoops good catch
